index.html

```diff
@@ -1,6 +1,12 @@
 <!DOCTYPE html>
 <html>
 <head>
+  <!--
+  "This is github!"
+  "No... THIS.  IS.  A WEBSITE"
```

Comment:
I can simply not accept this. Style guide clearly indicates that all comments must be in allcaps.